lib/bolt/cli.rb

```diff
@@ -99,6 +95,15 @@ def parse
 
       validate(options)
 
+      @config = if options[:configfile]
+                  Bolt::Config.from_file(options[:configfile], options)
+                else
+                  boltdir = options[:boltdir] ?  Bolt::Boltdir.new(options[:boltdir]) : Bolt::Boltdir.find_boltdir(Dir.pwd)
+                  Bolt::Config.from_boltdir(boltdir, options)
+                end
+
+      Bolt::Logger.configure(config.log, config.color)
```

Comment:
I almost want an `assert(@pal.nil?)` here to make sure we don't accidentally call `pal` before `@config` is finished. But tests will probably catch something that obvious.

Reply:
Yeah I think I'm comfortable letting tests handle that. Any of our tests that run the CLI should find it.